modules/stochastic_tools/test/tests/multiapps/dynamic_sub_app_number/tests

```diff
@@ -0,0 +1,24 @@
+[Tests]
+  issues = '#16842'
+  design = "Samplers/index.md SamplerFullSolveMultiApp.md"
+  [no_change]
+    requirement = "The system shall support generating random samples of data"
+    [constant]
+      type = JSONDiff
+      input = main.i
+      jsondiff = main_constant_size_out.json
+      allow_test_objects = true
+      cli_args = "Samplers/dynamic/increment_rows=0 Outputs/file_base=main_constant_size_out"
+
+      detail = "that remain constant in size and"
+    []
+    [dynamic]
```

Comment:
You might be still working on this. But it doesn't seem like you are testing running a sub-app with a dynamic sampler.

Reply:
The Sampler in main.i automatically adds a row with each call to execute, this is disabled in the "constant" one with the increment_row=0

Reply:
I understand that. I mean that you are not using the sampler to run a SamplerFullSolveMultiApp in any tests.

Reply:
I will add one.